ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -351,7 +353,11 @@ export default class ConfirmPageContainer extends Component {
             <PageContainerFooter
               onCancel={onCancel}
               cancelText={t('reject')}
-              onSubmit={isSetApproveForAll ? onSetApprovalForAll : onSubmit}
+              onSubmit={
+                isSetApproveForAll && isApprovalOrRejection
```

Comment:
For code reviewers:

`isApprovalOrRejection` is a variable that pre-dated this PR, but it is poorly named. It will be true for a`setApprovalForAll` transactions that, if approved, grant access to all NFTs in a collection. And it will be false for a `setApprovalForAll` transaction that revokes access.

In the former case, the transaction data looks like this:
```
Function: SetApprovalForAll
Parameters: true
Data: 0xa22cb4650000000000000000000000009bc5baf874d2da8d216ae9f137804184ee5afef40000000000000000000000000000000000000000000000000000000000000001
```

And in the latter:
```
Function: SetApprovalForAll
Parameters: false
0xa22cb4650000000000000000000000009bc5baf874d2da8d216ae9f137804184ee5afef40000000000000000000000000000000000000000000000000000000000000000
```

So this variable depends upon whether the approval is encoded within the last hexadecimal of the data. This is deciphered in the `helpers/utils/token-util` `getTokenApprovedParam` method which is called in `confirm-approve` to create the `isSetApproveForAll` variable, which is then passed down, eventually to this component

The goal of this PR was to ensure that the `isSetApproveForAll` warning popup is not shown when revoking. And so the solution is to use `isApprovalOrRejection` as part of the decision as to which method should be called upon clicking the confirmation button on the set approval for all screen. If `isApprovalOrRejection` is true, then the method that shows the popup is called, if it is false, then that method is not called, and instead the transaction is just approved/submitted.